PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -196,8 +200,15 @@ local function CreateCamera()
 	end
 
 	function this:RotateCamera(startLook, xyRotateVector)
+		local maxY = MAX_Y
```

Comment:
Can we also change this to not have to figure out which max/min y to use every rotation? I doubt vr enabled will be changing often, and we can listen to the changed event to handle when it does.